speedrun-3.html

```diff
@@ -99,7 +99,7 @@ <h3>Made by Ben Songster (MeemawHustlin) with help from the PRCLive community.</
 <li><a href="glitches.html#duping">Dupe</a> the 1 stack using the 2 stack, pick one up, then repeat and pick up all scrolls so that you have 3 of one scroll and 2 of another. Now, dupe the stacks back and forth until you have 512 of one and 768 of another.</li>
 <li>From the 768 stack, sell 200, 200, then 68 scrolls back to Calindil. From the 512 stack, sell 200, 200, then 62 scrolls. You should have 300 of one type, and 50 of another, as well as over 4,300 gold at the end. </li>
 <li>Read <span class="replaceable" clid="book87">The Black Arts on Trial [Mysticism]</span> skill book on the counter.</li>
-<li>Head SE to Red Diamond Jewelry. Talk to <span class="npc">Hamlof</span> and buy the Brass Ring. Dupe about 300 lockpicks.</li>
+<li>Head SE to Red Diamond Jewelry. Talk to <span class="npc">Hamlof Red-Tooth</span> and buy the Brass Ring. Dupe about 300 lockpicks.</li>
```

Comment:
Instead of spelling out his full name, add his formID as a clid here.
Hamlof isn't in npc.json yet, so you'll have to add an entry there as well.

Reply:
Can do. Is there a reason we're doing it this way?

Reply:
It's nice not to have to spell out an npc's entire name every time you want to reference them. It can make the guide easier to read.

Reply:
I think a better convention would be for the NPC names in the guide to match the names as they appear in the game

Reply:
I'm still not sure about it as a global policy, but as this is the first time in the guide that Hamlof is mentioned, he should probably have his full name here.